manifests/operator-ns-scoped.yaml

```diff
@@ -2896,7 +4986,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        image: qserv/qserv-operator:2022.1.1-rc1-15-gcc239da
+        image: qserv/qserv-operator:2022.1.1-rc1-58-ge596b7d-dirty
```

Comment:
Is it safe to rely on the `dirty` tag?

Reply:
Another comment - this file s really huge! It has nearly 5,000 lines. Does it need to be so big? Is there a possibility of splitting it into manageable files?